src/NHSD.BuyingCatalogue.Ordering.Api/ActionFilters/InputValidationActionFilter.cs

```diff
@@ -36,6 +36,8 @@ public void OnActionExecuting(ActionExecutingContext context)
         }
 
         public void OnActionExecuted(ActionExecutedContext context)
-        { }
+        {
+
```

Comment:
I don't think we need the blank line. I thought you were as fond of excess white space as I am 😀

Not exactly essential though, so up to you.

Reply:
hah damn resharper must have added it